kernel/safe_typing.mli

```diff
@@ -169,6 +169,12 @@ val add_module_parameter :
   MBId.t -> Entries.module_struct_entry -> Declarations.inline ->
     Mod_subst.delta_resolver safe_transformer
 
+(** returns the number of module parameters *)
+val module_num_parameters : safe_environment -> int
+
+(** returns true if the current module is a module type *)
+val module_is_modtype : safe_environment -> bool
```

Comment:
Should Lib.is_modtype be implemented in terms of this? Kinda weird to have both.